fluss-flink/fluss-flink-common/src/test/java/org/apache/fluss/flink/procedure/FlinkProcedureITCase.java

```diff
@@ -86,12 +87,42 @@ void before() throws ExecutionException, InterruptedException {
         tEnv.executeSql("use catalog " + CATALOG_NAME);
     }
 
+    @AfterEach
+    void after() throws Exception {
+        // Clean up any dynamic config changes made during the test
+        // to ensure tests don't affect each other
+        if (tEnv != null) {
+            try {
+                // Delete dynamic configs that might have been modified during tests
+                // This resets them to their initial static configuration values
+                tEnv.executeSql(
+                                String.format(
+                                        "Call %s.sys.set_cluster_config('%s')",
+                                        CATALOG_NAME,
+                                        ConfigOptions.KV_SHARED_RATE_LIMITER_BYTES_PER_SEC.key()))
+                        .await();
+                tEnv.executeSql(
+                                String.format(
+                                        "Call %s.sys.set_cluster_config('%s')",
+                                        CATALOG_NAME, ConfigOptions.DATALAKE_FORMAT.key()))
+                        .await();
+            } catch (Exception e) {
+                // Ignore cleanup errors to avoid masking test failures
+            }
+        }
```

Comment:
Can these be moved into test cases related to set cluster configuration? It feels a bit odd to place them in afterEach？